### ISSUE TITLE
smcs.rsp

### ISSUE THREAD

**Issue by sebas77** [SaladLab/Unity3D.IncrementalCompiler#15]
not really an issue, but are compiler settings still supported? If so, is it possible to have different compiler settings per project? A Unity solution is made of 4 projects, can I have different settings for each?

**Comment by veblush** [comment]
Yes, compiler settings are supported. But there is only one setting per unity project, which has been decided to follow how unity works. Why do you want to have separate settings?

**Comment by sebas77** [comment]
I need to disable warnings in my plugins folder. Code there can get obsolete and it's very annoying to see warnings of code we haven't written.

**Comment by veblush** [comment]
Yes, that is a common use for using compiler settings. Incremental compiler follows the way unity3d works as a settings per unity project (not sharp project).

**Comment by sebas77** [comment]
yes but I don't want to disable warnings for my project code, therefore I need the settings separated. I will post on Unity forum as well about it, I think it's needed as feature. Thanks for the reply.
